internal/commands/app/diff_test.go

```diff
@@ -139,7 +135,61 @@ Removed Dependencies
   - debug@4.3.1
 Modified Dependencies
   * underscore@1.9.1 -> underscore@1.9.2
-`, out.String())
+`
```

Comment:
extra newline here

Reply:
like the one deleted, the change message has a new line at the end, if I remove this and add `\n` at the end of `underscore@1.9.2`, it will be parsed as `\\n` in the actual message. Please see other tests in this file, I removed the line after ```.

Reply:
yup, my eyes were playing tricks with me late friday with "ignore whitespace" on... carry on here!